pkg/host/internal/network/network.go

```diff
@@ -95,10 +96,39 @@ func (n *network) TryGetInterfaceName(pciAddr string) string {
 		return name
 	}
 
-	log.Log.V(2).Info("tryGetInterfaceName()", "name", netDevName)
+	log.Log.V(2).Info("TryGetInterfaceName()", "name", netDevName)
 	return netDevName
 }
 
+// TryGetInterfaceName tries to find the SR-IOV virtual interface index base on pci address
+func (n *network) TryGetInterfaceIndex(pciAddr string) int {
+	ifName := n.TryToGetVirtualInterfaceName(pciAddr)
+	if ifName == "" {
+		return -1
+	}
+
+	netDir := filepath.Join(consts.SysBusPciDevices, pciAddr, "net", ifName)
+	if _, err := os.Lstat(netDir); err != nil {
```

Comment:
do we really need this check ?

between here and L118 the path may no longer be valid when netdev name changes via udev right after this spot.